tests/cert.json.rsa.in

```diff
@@ -0,0 +1,39 @@
+,
+    {"server_command": [@CHECKER@"openssl", "s_server", @PROPQ@"-www", "-port", "@PORT@", "-key", "@PRIURI@", "-cert", "@CRTURI@"],
+     "comment": "Run test with @PRIURI@ without certificate verify",
+     "environment": {"PYTHONPATH" : "."},
+     "server_hostname": "localhost",
+     "server_port": @PORT@,
+     "common_arguments": ["-p", "@PORT@"],
+     "tests" : [
+       {"name" : "test-tls13-conversation.py"},
+       {"name" : "test-conversation.py",
+        "arguments" : ["-d"]},
+       {"name" : "test-dhe-rsa-key-exchange-signatures.py",
+        "arguments" : [
+          "-n", "0",
+          "-x", "TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA sha1 signature", "-X", "handshake_failure",
+          "-x", "TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA sha224 signature", "-X", "handshake_failure",
+          "-x", "TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA sha256 signature", "-X", "handshake_failure",
+          "-x", "TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA sha384 signature", "-X", "handshake_failure",
+          "-x", "TLS_DHE_RSA_WITH_3DES_EDE_CBC_SHA sha512 signature", "-X", "handshake_failure",
+          "-e", "TLS_DHE_RSA_WITH_AES_128_CBC_SHA256 sha1 signature",
```

Comment:
why `-e` and not `-x`?

Reply:
My though was that by excluding the tests, we say "we do not care for the results", do not depend on particular openssl configuration and we could make it working on other distros/os than default fedora with default crypto policies that might allow SHA1 or brainpool. Right now CI runs also on Debian, Ubuntu, CentOS and MacOS.

Reply:
Sounds like here expecting failure works as expected, so resolving this one.

Reply:
no, I think the idea is to verify that SHA-1 actually _doesn't_ work, for that we need `-x`